tests/longevity-tests/get-and-start-harbor.bash

```diff
@@ -24,21 +24,20 @@ version=$1
     && pushd harbor && docker-compose start && popd && exit 0
 
 echo "Pulling down version ${version} of Harbor..."
-wget https://github.com/vmware/harbor/releases/download/v${version}/harbor-online-installer-v${version}.tgz -qO - | tar xz
+wget https://storage.googleapis.com/harbor-releases/harbor-online-installer-v${version}.tgz -qO - | tar xz
+
 echo "Configuring Harbor"
 sed -i 's/hostname = reg.mydomain.com/hostname = vic-executor1.eng.vmware.com/g' harbor/harbor.cfg
+
 echo "Installing & starting Harbor"
 sudo ./harbor/install.sh
 
-echo "Preparing Harbor..."
-echo "Logging in..."
+echo "Logging into Harbor"
 docker login vic-executor1.eng.vmware.com --username=admin --password="Harbor12345"
-echo "Pulling some images to put in Harbor and putting them in Harbor.."
 
-pushd tests/resources
-for image in $(python -c "vars=__import__('dynamic-vars'); print(\" \".join(vars.images))"); do
+echo "Pulling some images to put in Harbor and putting them in Harbor"
+for image in "busybox"; do
```

Comment:
Quick question, what is populating "busybox" here?  Where are we getting this value from to iterate through?

Reply:
it is static. that is currently the only image that longevity uses, if we ever add a different image it would just become
"busybox alpine"

Reply:
Awesome!  Thanks for explaining that.

Reply:
No, I don't like this. 

Is there a reason we don't want the same images that we use in CI? And now there are two seemingly-unrelated locations where the list needs to be updated, if we want Longevity and CI to test w/ the same images & versions?

If you must make this change, remove the `for` loop since there is only one element. The code is harder to grok w/ the for loop there, especially since there is only one element which leads folks to think "but there must be a for loop for a reason." "We might have more images in the future" is not an adequate reason to keep this for loop -- we're all capable of converting a set of instructions into a loop as necessary if we add more images to this list, it only takes a second, and otherwise we're just creating cruft.

Reply:
I will say that I'm not a huge fan of the hack I wrote that this is replacing _either_, really we should put testing image lists into something like a YAML file and then read from that in both the pybot code that the old code parses here as well as here, as long as it's advantageous to use the same images in both places, which perhaps it is not.